### DISCUSSION TITLE
More New Achievements Have Been Updated

### DISCUSSION THREAD

**Discussion by lnugraha** [Schweinepriester/github-profile-achievements#111] [General]
Today (July 13, 2021) I noticed that Github has just updated their achievement badges. I have identified the following new badges and how to earn them (please CMIIW)

1. Quickdraw: close an open issue (or pull request?) within 5 minutes; apparently one-shot like YOLO thing.

2. Pair Extraordinaire: coauthorized commits on merged pull request

3. Starstruck: earn at least 16 stars (from a single repository?) that you own

**Comment by Schweinepriester** [comment]
Oh wow, hello there!

Thank you very much for letting me know, I will get right to it!

Do you example links for each?

**Reply by lnugraha** [reply]
It appears that there are another three new achievements, although I only know how to unlock two of them. It is also unclear whether the previous three badges are still obtainable.

1. **Quickdraw:** one shot

2. **Pair Extraordinaire:**
- [ ] ??? to unlock
- [ ] ??? to bronze
- [ ] ??? to silver
- [ ] ??? to gold

3. **Starstruck:**
- [ ] 16 stars to unlock
- [ ] 128 stars to bronze
- [ ] 512 stars to silver
- [ ] 4096 stars to gold

**Reply by Schweinepriester** [reply]
On it!
- https://github.com/Schweinepriester/github-profile-achievements/issues/112
- https://github.com/Schweinepriester/github-profile-achievements/pull/113

**Comment by nmcardoso** [comment]
can I get the **Pair Extraordinaire** badge if I merge my own pr co-authored with others?
